
# pre pick and post pick params [x, y, z] wrt robot base frame
pre_pick_offset: [-0.1, 0, 0]
pre_place_offset: [-0.4, 0, 0.1]

# pre-defined middle poses in joint states (rad)
## left
pre_middle_left: [0.8, -1.7568, 2.5506, 0.7786, -1.5734, -0.0038]
# post_middle_left: [1.0014, 0.7062, 1.9060, -0.0747, -0.0039, -0.0804]
post_middle_left: [1.3014, 0.7062, 1.9060, -0.0747, -0.0039, -0.0804]

## right
pre_middle_right: [0.8, 1.7568, -2.5506, -0.7786, 1.5734, 0.0038]
# post_middle_right: [0.7015,-0.4631,-1.6642, -1.0472, 0.0081, -0.0615]
post_middle_right: [1.3014, -0.7062, -1.9060, 0.0747, 0.0039, 0.0804]

# pre-defined place poses in joint states (rad)
## left
place_left: [0.9706, 1.4689, 1.9270, -0.2326, 0.0, 0.0]
## left
place_right: [0.9706, -1.4689, -1.9270, 0.2326, 0.0, 0.0]

# Parallel running parameters
detected_obj_pose_position: [0,0,0]
detected_obj_pose_orientation: [0,0,0,0]